mri/utils/process_SWI

```diff
@@ -0,0 +1,127 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+#functions
+def virtual_coil_reconstruction(imgs):
+    """
+    Calculate the combination of all the coils thanks to the virtual coil
```

Comment:
Thanks to virtual coil? I am not sure if this is a typo.
`Calculate the combination of all coils using virtual coil reconstruction` could be used?